src/sql/parts/query/editor/messagePanel.ts

```diff
@@ -211,6 +282,14 @@ export class MessagePanel extends ViewletPanel {
 		dispose(this.queryRunnerDisposables);
 		super.dispose();
 	}
+
+	public get selectAllMessages(): boolean {
```

Comment:
it's a little confusing to have different variables `selectAllMessages`, one a `string[]` and the other a `boolean`.  Perhaps the boolean could be named `shouldSelectAllMessages` or something like that to disambiguate.